privacyidea/static/components/login/controllers/loginControllers.js

```diff
@@ -417,7 +417,10 @@ angular.module("privacyideaApp")
             $scope.backend_debug_passwords = data.result.value.debug_passwords;
             $scope.privacyideaVersionNumber = data.versionnumber;
             var lang = gettextCatalog.getCurrentLanguage();
-            $scope.privacyideaSupportLink = "https://netknights.it/" + lang + "/support-link-" + data.result.value.role;
+            if (data.result.value.hasOwnProperty("supportmail"))
```

Comment:
please use `{ }` even for one-liners to keep it consistent

Reply:
Will do

Reply:
done in 849b6aec9